frontend/src/pages/detail/components/DetailReview/DetailReview.tsx

```diff
@@ -1,41 +1,43 @@
 import styled from '@emotion/styled';
 import filledStar from '../../../../common/assets/images/starIcon.svg';
 import ReviewItem from '../ReviewItem/ReviewItem';
+import { getReviews } from '../../apis/getReviews';
+import { useEffect, useState } from 'react';
+import { ReviewResponse } from '../../types/ReviewResponse';
 
-const REVIEW_MOCK = {
-  ratingAverage: 4,
-  ratingCount: 2,
-  reviews: [
-    {
-      id: 23,
-      reviewerName: '김**',
-      createdAt: '2025-08-07',
-      rating: 5,
-      content:
-        '정말 전문적이고 친절한 상담이었습니다. 개인별 맞춤 운동법을 자세히 설명해주셔서 매우 도움이 되었어요. 15분이 짧다고 생각했는데 알찬 내용으로 가득했습니다!',
-    },
-    {
-      id: 36,
-      reviewerName: '박**',
-      createdAt: '2025-08-07',
-      rating: 3,
-      content: '보통이었어요',
-    },
-  ],
-};
+interface DetailReviewProps {
+  mentoringId: number;
+}
+
+function DetailReview({ mentoringId }: DetailReviewProps) {
+  const [reviews, setReviews] = useState<ReviewResponse | null>(null);
+
+  const fetchReview = async () => {
+    try {
+      const response = await getReviews(mentoringId);
+      setReviews(response);
+    } catch (error) {
+      console.error('Error fetching reviews:', error);
+    }
+  };
+  useEffect(() => {
+    fetchReview();
+  }, []);
+  console.log(reviews);
+
+  if (!reviews) return null;
 
-function DetailReview() {
   return (
     <StyledContainer>
       <StyledTotalWrapper>
         <img src={filledStar} />
-        <strong>{REVIEW_MOCK.ratingAverage}</strong>
-        <p>({REVIEW_MOCK.ratingCount}개 리뷰)</p>
+        <strong>{reviews.ratingAverage}</strong>
+        <p>({reviews.ratingCount}개 리뷰)</p>
       </StyledTotalWrapper>
       <StyledReviewList>
-        {REVIEW_MOCK.reviews.map((review) => (
-          <ReviewItem key={review.id} review={review} />
-        ))}
+        {reviews.reviews.map((review) => {
+          return <ReviewItem key={review.id} review={review} />;
+        })}
```

Comment:
p4: map 내부에서 추가적인 로직이 없어서 바로 ()로 return 해도 될 거 같아! 

```suggestion
        {reviews.reviews.map((review) => 
<ReviewItem key={review.id} review={review} />
        )}
```

Reply:
[refactor: 불필요한 return 제거](https://github.com/woowacourse-teams/2025-Fit-toring/pull/433/commits/8c751e82c6aa307f7de52cea3a7e0e2a600b7561)